src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.container.js

```diff
@@ -33,13 +45,17 @@ export const WidgetEnrollment = ({ teiId, enrollmentId, programId, onDelete, onA
         <WidgetEnrollmentComponent
             enrollment={enrollment}
             canAddNew={canAddNew}
+            editDateEnabled={!readOnlyMode && program && program.access.data.write}
+            displayAutoGeneratedEventWarning={containsAutoGeneratedEvent}
             program={program}
             refetchEnrollment={refetchEnrollment}
             refetchTEI={refetchTEI}
             ownerOrgUnit={{ id: ownerOrgUnit, displayName }}
             loading={!(enrollment && program && displayName)}
             onDelete={onDelete}
             onAddNew={onAddNew}
+            onUpdateEnrollmentDate={onUpdateEnrollmentDate}
+            onUpdateIncidentDate={onUpdateIncidentDate}
             error={error}
```

Comment:
can we change this to `initError`? (hard to keep track of the errors now when we also have error callbacks when updating)